test/functionalTest/cases_xml/0722_unhardwire_format_cpr/query_forward_in_json.test

```diff
@@ -1,249 +0,0 @@
-# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
I understand that all .test under 0722_unhardwire_format_cpr are testing the `notifyFormat` feature to override default format, which is not longer supported as we have only one format (JSON).